src/components/OptionsSelector.js

```diff
@@ -191,11 +191,16 @@ class OptionsSelector extends Component {
             CTRLEnterConfig.shortcutKey,
             () => {
                 const focusedOption = this.state.allOptions[this.state.focusedIndex];
-                if (!this.canSelectMultipleOptions && !focusedOption) {
+                if (this.props.canSelectMultipleOptions) {
+                    this.props.onConfirmSelection(focusedOption);
                     return;
                 }
 
-                this.props.onConfirmSelection(focusedOption);
+                if (!focusedOption) {
+                    return;
+                }
```

Comment:
Should this be above the first check since the code inside that block is also using the `focusedOption`?

Reply:
Moving this up as you've suggested breaks things, because it prevents `CMD+Enter` from creating a group chat if there are participants selected and the search bar is highlighted. But as it turns out, `onConfirmSelection` doesn't really need the `focusedOption` and that's not really it's intended use. I did an audit of all of it's usages:

- [here](https://github.com/Expensify/App/blob/daff15b5ef0167d9455a756f0c3f4d553fa8acc3/src/pages/NewChatPage.js#L258) it's only ever triggered for group chats, and `createGroup` takes no arguments. So I clarified that and replace the 1:1 `createChat` method with a noop.
- [here](https://github.com/Expensify/App/blob/cf6bb2a0c12baf5183423fb39c64fc7ae4a958af/src/pages/workspace/WorkspaceInvitePage.js#L286) invite user takes no arguments
- [here](https://github.com/Expensify/App/blob/b687fcecd3fa546403bf057f0287f8f46197a317/src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsSplit.js#L232) `finalizeParticipants` takes no arguments.
- [here](https://github.com/Expensify/App/blob/b01d1ac45d83f5a271a6f57fcd6b5c69080047f7/src/components/IOUConfirmationList.js#L351) `confirm` take an argument that's only used by the `sendMoney` flow. And in the `sendMoney` flow `onConfirmSelection` is never called. I looked into splitting `confirm` into `confirmSend` and `confirmRequest`, but it feels out-of-scope and everything seems to be working as expected.

Reply:
Good eye though!

Reply:
Ok cool.